### ISSUE TITLE
[MNG-6213] Maven doesn't check the validity of scope value

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#7234]
**[Jin Kwon](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jinahya)** opened **[MNG-6213](https://issues.apache.org/jira/browse/MNG-6213?redirect=false)** and commented

I accidentally  put a wrong value into the `scope` tag.

```xml
<dependencyManagement>
  <dependencies>
    <dependency>
      <groupId>org.glassfish.jersey</groupId>
      <artifactId>jersey-bom</artifactId>
      <version>2.26-b03</version>
      <type>pom</type>
      <scope>include</scope> <!-- should be 'import' -->
    </dependency>
  </dependencies>      
</dependencyManagement>
```

And, of course, my dependency doesn't work.

```xml
<dependencies>
  <dependency>
    <groupId>org.glassfish.jersey.core</groupId>
    <artifactId>jersey-common</artifactId>
    <scope>test</scope>
  </dependency>
</dependencies>
```

```
'dependencies.dependency.version' for org.glassfish.jersey.core:jersey-common:jar is missing.
```

Maven should complains about the wrong value of `include`.

---

**Affects:** 3.3.9

**Remote Links:**
- [GitHub Pull Request #131
](https://github.com/apache/maven/pull/131)
- [GitHub Pull Request #131
](https://github.com/apache/maven/pull/131)

0 votes, 6 watchers

**Comment by jira-importer** [comment]
**[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise)** commented

The problem by checking the scope is simply this would break many people cause they are using different scopes (I can remember about JavaFX is using a different scope etc.)

**Comment by jira-importer** [comment]
**[Michael Warnecke](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=mwarnecke)** commented

Are different scopes only used in `dependencyManagement`? Because there ist a check in place for the `dependency` section

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise), is it still an issue merging this one? It seems to make sense.Specially because the PR is set to a warning, not an error.

**Comment by jira-importer** [comment]
**[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise)** commented

Ok lets try it...

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [9c946263facbfa851fb646b3fac62cf1d3302ded](https://gitbox.apache.org/repos/asf?p=maven.git;a=commit;h=9c946263facbfa851fb646b3fac62cf1d3302ded).
